Gordon360/AuthorizationFilters/StateYourBusiness.cs

```diff
@@ -420,6 +444,29 @@ private bool canAdd(string resource)
                         return true;
                     else
                         return false; // Only super admin can add Advisors through this API
+                case Resource.HOUSING_ADMIN:
+                    {
+                        if (user_position == Position.SUPERADMIN)
+                            return true;
```

Comment:
Technically, this is a useless check since superadmins are cleared without ever checking the operation against the resource - they are automatically for everything.

Reply:
Would that mean removing this HOUSING_ADMIN section from ADD entirely, or would we remove this `if` and just leave it with a `return false` so that it rejects non-admins by default?

Reply:
Well the `HOUSING_ADMIN` resource is very clear and explicit so I think that should stay, especially in case the authorization logic changes in the future. I was thinking of leaving the `HOUSING_ADMIN` resource in but having it `return false` by default, potentially with a comment to the effect of `only superadmins can [add/delete/update/etc] a HOUSING_ADMIN`. 

This is exactly the way that the `ADMIN` resource currently works, so it has precedent in the codebase. Although it is technically repetitive to have both with the same logic, they represent different resources so it's important to distinguish them, especially if either needs to change authorization logic.